data/gui/window/statistics_dialog.cfg

```diff
@@ -177,41 +177,105 @@
+                                                    id = {colid_overall}
+                                                    definition = "default_small"
+                                                    tooltip = {ratio_tooltip}
+                                                [/label]
+                                            [/column]
+
+                                            [column]
+                                                grow_factor = 0
+                                                border = "all"
+                                                border_size = 5
+                                                horizontal_grow = false
+
+                                                [label]
+                                                    id = "overall_percent"
+                                                    definition = "default_small"
+                                                    text_alignment = "right"
+                                                    use_markup = true
+                                                    tooltip = {percent_tooltip}
+                                                [/label]
+                                            [/column]
+
+                                            [column]
+
+                                                # Spacer to make up for showing only one percentage, not two as in the next table
+                                                [spacer]
+                                                    width=10 # TODO: Wild guess. Feel free to adjust
+                                                [/spacer]
+                                            [/column]
+
+                                        [/row]
+                                    [/grid]
+                                [/column]
+
+                                [column]
+                                    grow_factor = 1
+                                    border = "all"
+                                    border_size = 0
+                                    horizontal_grow = true
+                                    [grid]
+                                        linked_group = "cost"
+                                        [row]
+
+                                            [column]
+                                                grow_factor = 1
+                                                border = "all"
+                                                border_size = 5
+                                                horizontal_grow = true
+
+                                                [label]
+                                                    id = {colid_this_turn}
+                                                    definition = "default_small"
+                                                    tooltip = {ratio_tooltip}
+                                                [/label]
+                                            [/column]
```

Comment:
Mawmoocn on the forums asks about vertical alignment between the hits and damage tables and between the percentage values in each table.

Reply:
That has been fixed already, hasn't it?